scripts/publish.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
```

Comment:
I'm assuming this file has not changed significantly from the one we reviewed together.

Reply:
Correct.